.obs/workflows.yml

```diff
@@ -22,6 +22,10 @@ pr:
           target_repository: openSUSE_Leap_15.5
         - target_project: devel:openQA
           target_repository: openSUSE_Leap_15.5
+      - name: SLE_15_SP6_Backports
+        paths:
+        - target_project: openSUSE:Backports:SLE-15-SP6:Update
+          target_repository: standard
         architectures: [ x86_64 ]
```

Comment:
This needs to be added as well. You are basically removing the architecture from the previous repository

Reply:
Done